Wrapping/R/CMakeLists.txt

```diff
@@ -91,6 +91,33 @@ if(NOT SimpleITK_FORBID_DOWNLOADS)
     )
 
 endif()
+# Download tar of Rd files
+if(NOT SimpleITK_FORBID_DOWNLOADS)
+  include(sitkExternalData)
```

Comment:
I separated as the actions after download are different - copy vs tar extraction. Is there a good way to separate the two if the download code is common? Using filenames seems just as complicated, but feasible